Automattic/govpack
# feat(autoload): moved CMB2 to use autoloader

### All Submissions:



### Changes proposed in this Pull Request:

1. Adds CMB2 as a Composer dependency
2. Tells composer to auto load CMB2
3. Removes manual calling and loading of CMB2
4. Removed Files associated with CMB2 that where in the repo



### How to test the changes in this Pull Request:

1. Checkout branch
2. Double check the CMB2 folder is removed from your local file system
3. Try load a Profile form, will fail
4. Run `composer update`
5. When complete try load a profile page, should load correctly
